tests/testthat/test-filter.R

```diff
@@ -0,0 +1,229 @@
+# TODO: implement these functions in filters.R and use them in this test
+include <- function(...) {}
+exclude <- function(...) {}
+lessthan <- function(...) {}
+morethan <- function(...) {}
+interval <- function(...) {}
+
+data(original_data)
+original_samples_count <- nrow(original_data$data)
+
+test_cases <- list()
+
+test_cases[[length(test_cases) + 1]] <- list(
```

Comment:
maybe append(test_cases, list(data_factory = function ..........) is more readable?

Reply:
`append` will merge 2 lists, and I need to add a list as item to test_cases list